src/main/java/rife/scheduler/taskoptionmanagers/DatabaseTaskOptions.java

```diff
@@ -179,7 +179,7 @@ protected boolean _removeTaskOption(Delete removeTaskOption, final int taskId, f
         return result;
     }
 
-    protected class ProcessTaskOption extends DbRowProcessor {
+    protected static class ProcessTaskOption extends DbRowProcessor {
```

Comment:
This should remain non static in case someone extends it and wants access to the instance